open-policy-agent/README.md

```diff
@@ -83,3 +83,85 @@ For example, applying 'kubeadmin' temporary user removal validation policy
 $ oc apply -f https://raw.githubusercontent.com/openshift-4-compliance/openshift-4-compliance-automation/master/open-policy-agent/authentication-user-management/delete-kubeadmin/template.yaml
 $ oc apply -f https://raw.githubusercontent.com/openshift-4-compliance/openshift-4-compliance-automation/master/open-policy-agent/authentication-user-management/delete-kubeadmin/constraint.yaml
 ```
+
+## Applying policies using GitOps
+You can include in `dryrun` mode either all policies or as policy groups.
+You can change to `enforce` mode either all policies, as policy groups, or individually.
```

Comment:
Change from `You can change to `enforce` mode either all policies, as policy groups, or individually.` to `You can use the `deny` enforcement mode in either all policies, in policy groups, or individually.`